### PR TITLE
Completed Warmup Exercise.

### PR DESCRIPTION
**Pull request by Aditi-Sawardekar**
Initial Commit

### CODE REVIEW

File: nodejs/week1/avg.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const getNumbers = (process.argv[2]);
+if (getNumbers === undefined){
+    console.log("Enter numbers")
+    return
```
Comment (itsgauravsaxena) [inline nodejs/week1/avg.js]: empty return here seems to be irrelevant. May be best to remove it

**Review by itsgauravsaxena — COMMENTED** [review]
Just a small comment else it looks good!

File: nodejs/week1/avg.js (hunk 1 above)
Comment (Aditi-Sawardekar) [reply nodejs/week1/avg.js]: Hi Gaurav,
Thanks for reviewing my homework. I have made the suggested change. Please approve the merge request.

**Review by itsgauravsaxena — APPROVED** [review]

**Comment by Aditi-Sawardekar** [comment]
Hi Gaurav,
Thanks for reviewing and approving the changes.
